### PR TITLE
#95 - Implement InMemoryContributorsTestCase

### PR DESCRIPTION
**Pull request by HoreaPetrila**
PR For #95

### CODE REVIEW

File: self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryContributorsTestCase.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    @Test
+    public void contributorGetById() {
+        final Storage storage = new InMemory();
+        Contributor registered = storage.contributors()
+            .register("horea", "github");
+        assertThat(storage.contributors()
+            .getById("horea", "github"),
+            is(registered));
+
+    }
```
Comment (amihaiemil) [inline self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryContributorsTestCase.java:45]: @HoreaPetrila Can you also add a unit test to check that ``getById`` returns ``null`` when the Contributor does not exist?

**Comment by amihaiemil** [comment]
@HoreaPetrila looks good, thanks!

**Comment by amihaiemil** [comment]
@rultor merge it
